cpp/open3d/t/geometry/PointCloud.h

```diff
@@ -334,11 +334,24 @@ class PointCloud : public Geometry, public DrawableGeometry {
                              bool remove_duplicates = false) const;
 
     /// \brief Downsamples a point cloud with a specified voxel size.
+    ///
     /// \param voxel_size Voxel size. A positive number.
     PointCloud VoxelDownSample(double voxel_size,
                                const core::HashBackendType &backend =
                                        core::HashBackendType::Default) const;
 
+    /// \brief Downsamples a point cloud uniformly
```

Comment:
please elaborate on the description such as:
Downsamples a point cloud, by selecting every kth index point and its attributes.

Similarly for RandomDownSample

Reply:
Done